secure_message/resources/threads.py

```diff
@@ -17,12 +17,13 @@ class ThreadById(Resource):
     @staticmethod
     def get(thread_id):
         """Get messages by thread id"""
-        string_query_args, page, limit, ru_id, survey, cc, label, desc, ce = get_options(request.args)
+        string_query_args, page, limit, _, _1, _2, _3, _4, _5 = get_options(request.args)  # NOQA TODO - Named Tuple
```

Comment:
Are these args documented anywhere?  I think they are but wanted to double check because the variable names acted like documentation (even though they probably fail pylint for not being used)